### DISCUSSION TITLE
Welcome to portfolio Discussions!

### DISCUSSION THREAD

**Discussion by futurestar425** [futurestar425/portfolio#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by futurestar425** [comment]
How to deploy on onrender.com?

**Comment by max0700** [comment]
First, you have to login on onrender.com

**Reply by futurestar425** [reply]
good.
